cmd/ttn-lw-cli/commands/users.go

```diff
@@ -339,6 +341,7 @@ func init() {
 	usersUpdatePasswordCommand.Flags().AddFlagSet(userIDFlags())
 	usersUpdatePasswordCommand.Flags().String("old", "", "")
 	usersUpdatePasswordCommand.Flags().String("new", "", "")
+	usersUpdatePasswordCommand.Flags().Bool("revoke_all_access", false, "revoke all sessions and access tokens after the password is updated")
```

Comment:
Shouldn't this be `true` by default?

Reply:
As I understood the issue, 

> Users should get the possibility to revoke all User Sessions and OAuth Access Tokens on a password change.

I thought this should be optional? cc @htdvisser

Reply:
I think it's fine to leave this optional. There is no reason to revoke existing tokens unless you suspect that your credentials were compromised.

Reply:
I think this has to be `true`.

I have yet to see the first (corporate) platform where a password change does not lead to a signout everywhere. Even in consumer services it works this way; think Gmail, iCloud, etc.

Reply:
And yes this also goes for OAuth client. If you change your Facebook password, you will go through the consent screens of third-party apps again.

Reply:
I'm fine with either. I guess that most users will use the `--help` on this command and consider whether they need the flag anyway.

Reply:
> I guess that most users will use the `--help` on this command and consider whether they need the flag anyway.

I guess not. Let's change this default to a more secure default

Reply:
so then from this logic the flag would be `--do-not-revoke-access` or something like this?

Reply:
We'd just make the default `true`

BTW, this is a flag, why are we using snake case and not dashes?

Reply:
The flag is still `--revoke-all-access`, but it's `true` by default. You can set it to `false` with `--revoke-all-access=false`

Reply:
> BTW, this is a flag, why are we using snake case and not dashes?

I guess I confused this with the way the error messages are defined.

I will open and take over the issue to fix this, both with the default value and snake case.

Reply:
All flags are normalized by the CLI, so flags with `_` and `-` are equivalent. I think we're mixing them in more places.

Reply:
Shall change it anyway since I will bring other changes in the same line of code, for more consistency?

I see way more dashes in the cli flags definitions, so why not bring a bit more consistency

Reply:
>I think we're mixing them in more places.

Which one? Looks like this is the first

> Shall change it anyway since I will bring other changes in the same line of code, for more consistency?

Yes please

Reply:
> I see way more dashes in the cli flags definitions, so why not bring a bit more consistency

Be careful, some of those may be used in field masks that expect underscores.